alphastats/llm/uniprot_utils.py

```diff
@@ -270,6 +291,7 @@ def get_info(genes_list: List[str], organism_id: str) -> List[str]:
 
     for gene in genes_list:
         result = get_uniprot_data(gene, organism_id)
```

Comment:
(nit) `results` (also below)

Reply:
This funciton disappeared later, but I left a TODO to chekc for other instances.